setup.py

```diff
@@ -147,6 +147,7 @@
         celery=[
             'celery>=3.1.25.0,<4.4.7;python_version<="3.5"',
             'celery>=4,<5.3;python_version>"3.5"',
+            'kombu<5.2.0',
```

Comment:
It looks like kombu<6 are compatible with celery 5.1.2: https://github.com/celery/celery/blob/v5.1.2/requirements/default.txt#L3

And it also looks like the errors were coming from the `redis` client. Did you try bumping the `redis` package version in requirements.tests.txt instead?

Reply:
Determined the minimum version of redis that supports the username parameter and added that to the `requirements.test.txt` file